lib/ansible/modules/cloud/amazon/elasticache_snapshot.py

```diff
@@ -234,7 +231,8 @@ def main():
 
 # import module snippets
 from ansible.module_utils.basic import *
-from ansible.module_utils.ec2 import *
+from ansible.module_utils.ec2 import boto3_conn, get_aws_connection_info, ec2_argument_spec, camel_dict_to_snake_dict
```

Comment:
These should be towards the top, just below the end of the documentation sections